src/index.html

```diff
@@ -1,13 +1,458 @@
+              <source srcset="./images/products/bose-portable-mobile.png">
+              <img
+                class="product-photo"
+                src="./images/products/bose-portable-mobile.png"
+                alt="Bose"
+              >
+            </picture>
+            <a href="#" class="product-link">
+              <h3 class="product-title">Bose portable Smart speaker</h3>
+            </a>
+            <p>Smart home</p>
+            <h3>$ 399.00</h3>
+          </article>
+
+          <article class="product">
+            <picture>
+              <source srcset="./images/products/sound-link-flex-desktop.png" media="(min-width: 768px)">
+              <source srcset="./images/products/sound-link-flex-mobile.png">
+              <img
+                class="product-photo"
+                src="./images/products/sound-link-flex-mobile.png"
+                alt="Bose"
+              >
+            </picture>
+            <a href="#" class="product-link">
+              <h3 class="product-title">SoundLink Flex Bluetooth speaker</h3>
+            </a>
+            <p>Portable bluetooth</p>
+            <h3>$ 149.00</h3>
+          </article>
+
+          <article class="product">
+            <picture>
+              <source srcset="./images/products/sound-link-color-desktop.png" media="(min-width: 768px)">
+              <source srcset="./images/products/sound-link-color-mobile.png">
+              <img
+                class="product-photo"
+                src="./images/products/sound-link-color-mobile.png"
+                alt="Bose"
+              >
+            </picture>
+            <a href="#" class="product-link">
+              <h3 class="product-title">SoundLink Color Bluetooth speaker II</h3>
+            </a>
+            <p>Portable bluetooth</p>
+            <h3>$ 129.00</h3>
+          </article>
+        </div>
+      </section>
+
+      <section class="page__section categories" id="categories">
+        <h2 class="page__section-title">
+            Browse Bose
+          <br>
+            products by category
+        </h2>
+
+        <div class="container">
+          <div class="categories__type type">
+            <div class="type__photos">
+              <div class="grid">
+                <div class="
+                  grid__item
+                  grid__item--tablet-1-4
+                  grid__item--desktop-1-8
+
+                ">
+                  <picture>
+                    <source srcset="./images/galerey/desktop/1.png"  media="(min-width: 1024px)">
+                    <source srcset="./images/galerey/tablet/1.png"  media="(min-width: 744px)">
+                    <source srcset="./images/galerey/mobile/1.png" media="(min-width: 743px)">
+                    <img
+                      class="type__photo"
+                      src="./images/galerey/mobile/1.png"
+                      alt="Bose"
+                    >
```

Comment:
Something wrong with the srcset. Images on tablets are of low quality. I think you need to use images that are fit to upper breakpoint levels instead of down: f.e. for mobile use images that are ok on a width of 743px (in your case you have chosen minimum).

![Bose](https://user-images.githubusercontent.com/54940595/220616679-2126bc43-cb7a-48bb-80f8-58eb148d1be0.png)